### DISCUSSION TITLE
Do you have intent to add support for CloudFoundry provider?

### DISCUSSION THREAD

**Discussion by ramacobr** [sablierapp/sablier#262] [Q&A]
It's a very intersting ideia cause now a day exists much legacy environments running based on CloudFoundry. And Recently traefik disable the very useful plugin that solved the same issue that Sablier.

**Comment by acouvreur** [comment]
You can open an issue about the CloudFoundry provider.
You will be asked to provde resources to help people understand if that looks feasible.

If it is and it has traction, people will most likely implement it :)
